tests/integrationtest/l4_service_test.go

```diff
@@ -779,6 +779,39 @@ func TestWithInfraSettingStatusUpdates(t *testing.T) {
 		setting, _ := lib.GetCRDClientset().AkoV1alpha1().AviInfraSettings().Get(context.TODO(), settingName, metav1.GetOptions{})
 		return setting.Status.Status
 	}, 15*time.Second).Should(gomega.Equal("Rejected"))
+	g.Expect(nodes[0].VSVIPRefs[0].NetworkNames[0]).Should(gomega.Equal("net123"))
+
+	settingUpdate = (FakeAviInfraSetting{
+		Name:        settingName,
+		SeGroupName: "thisisaviref-seGroup",
+		Networks:    []string{"multivip-network1", "multivip-network2", "multivip-network3"},
+		EnableRhi:   true,
+	}).AviInfraSetting()
+	settingUpdate.ResourceVersion = "4"
+	if _, err := lib.GetCRDClientset().AkoV1alpha1().AviInfraSettings().Update(context.TODO(), settingUpdate, metav1.UpdateOptions{}); err != nil {
+		t.Fatalf("error in updating AviInfraSetting: %v", err)
+	}
+
+	g.Eventually(func() string {
+		setting, _ := lib.GetCRDClientset().AkoV1alpha1().AviInfraSettings().Get(context.TODO(), settingName, metav1.GetOptions{})
+		return setting.Status.Status
+	}, 15*time.Second).Should(gomega.Equal("Accepted"))
+
+	g.Eventually(func() string {
+		if found, aviModel := objects.SharedAviGraphLister().Get(SINGLEPORTMODEL); found && aviModel != nil {
+			if nodes := aviModel.(*avinodes.AviObjectGraph).GetAviVS(); len(nodes) > 0 {
+				return nodes[0].ServiceEngineGroup
+			}
+		}
+		return ""
+	}, 35*time.Second).Should(gomega.Equal("thisisaviref-seGroup"))
+	_, aviModel = objects.SharedAviGraphLister().Get(SINGLEPORTMODEL)
+	nodes = aviModel.(*avinodes.AviObjectGraph).GetAviVS()
+	g.Expect(len(nodes[0].VSVIPRefs[0].NetworkNames)).Should(gomega.Equal(3))
+	g.Expect(nodes[0].VSVIPRefs[0].NetworkNames[0]).Should(gomega.Equal("multivip-network1"))
+	g.Expect(nodes[0].VSVIPRefs[0].NetworkNames[1]).Should(gomega.Equal("multivip-network2"))
+	g.Expect(nodes[0].VSVIPRefs[0].NetworkNames[2]).Should(gomega.Equal("multivip-network3"))
+	g.Expect(*nodes[0].EnableRhi).Should(gomega.Equal(true))
```

Comment:
could we also add this test case for L7, using Ingress/IngressClass.

Reply:
added the tests for ingress class